packages/components/avatar/README.mdx

```diff
@@ -22,8 +22,8 @@ import { Avatar, Size, Variant } from '@contentful/f36-avatar';
 
 The Avatar can be rendered in different variants:
 
-- **User** - default variant
-- **App** - default variant
+- **user** - default variant
```

Comment:
Let's keep the lists as they were for the docs website, it looks better and I don't think we need the literal values here, they are listed in both examples and the prop table.